test/e2e/framework/util.go

```diff
@@ -2429,6 +2432,7 @@ func dumpAllNodeInfo(c clientset.Interface) {
 	DumpNodeDebugInfo(c, names, Logf)
 }
 
+// DumpNodeDebugInfo dumps debug information of given nodes.
```

Comment:
Thanks @atoato88 
added a couple of minor comments.
```suggestion
// DumpNodeDebugInfo dumps debug information of the given nodes.
```

Reply:
done.